http/src/request/validate.rs

```diff
@@ -1175,6 +1177,23 @@ fn _sticker_tags(value: &str) -> bool {
         .contains(&len)
 }
 
+pub fn communication_disabled_until(timestamp: Timestamp) -> bool {
+    _communication_disabled_until(timestamp)
+}
+
+const COMMUNICATION_DISABLED_MAX_DURATION: i64 = 28 * 24 * 60 * 60;
+
+#[allow(clippy::cast_possible_wrap)] // casting of unix timestamp should never wrap
+fn _communication_disabled_until(timestamp: Timestamp) -> bool {
```

Comment:
I can't find a better way to do this, but I'd at least like to avoid the unwrap by returning false if `SystemTime::duration_since` fails.